quinn-proto/src/connection/timer.rs

```diff
@@ -54,7 +57,7 @@ impl TimerTable {
         self.data.iter().filter_map(|&x| x).min()
     }
 
-    pub fn is_expired(&self, timer: Timer, after: Instant) -> bool {
-        self.data[timer as usize].map_or(false, |x| x <= after)
+    pub fn is_expired(&self, timer: Timer, after: Instant, default_none: bool) -> bool {
```

Comment:
The new argument here is detrimental to readability. Instead, let's remove `#[cfg(test)]` from `pub fn get` above and use that when more specialized behavior is required.

Reply:
Ack. :-) I remember that i changed between these two ways several times